debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/UnwrapFromMongoDbEnvelopeTest.java

```diff
@@ -39,6 +42,8 @@
 
     private static final String SERVER_NAME = "serverX.";
     private static final String PREFIX = SERVER_NAME + ".";
+    private static final String FLATTEN_STRUCT = "flatten.struct";
+    private static final String DELIMITER = "delimiter";
```

Comment:
I'd suggest to name it "flatten.delimiter" "flatten.struct.delimiter" to make its meaning clearer.

Reply:
I will make these changes by the next commit.